content/en/docs/alerts/_index.md

```diff
@@ -178,3 +178,62 @@ Here's the same output, pretty-printed:
 ## gRPC Output
 
 If you'd like to send alerts to an external program connected via gRPC API (for example, the [falco-exporter](https://github.com/falcosecurity/falco-exporter)), you need to enable both the `grpc` and `grpc_output` options as described under the [gRPC Configuration section](/docs/grpc/#configuration).
+
+
+{{% pageinfo color="primary" %}}
+
+_SELinux note_: If you have [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) enabled you may see the following error while starting Falco with gRPC support enabled:
+
+```shell
+
+May 08 12:14:41 myserver falco[15442]: Falco version X.XX.X (driver version ZZZZ)
+May 08 12:14:41 myserver falco[15442]: Falco initialized with configuration file /etc/falco/falco.yaml
+May 08 12:14:41 myserver falco[15442]: Loading rules from file /etc/falco/falco_rules.yaml:
+May 08 12:14:42 myserver falco[15442]: Loading rules from file /etc/falco/falco_rules.local.yaml:
+May 08 12:14:42 myserver falco[15442]: Loading rules from file /etc/falco/k8s_audit_rules.yaml:
+May 08 12:14:43 myserver falco[15442]: Starting internal webserver, listening on port 8765
+May 08 12:14:43 myserver falco[15442]: gRPC server threadiness equals to 2
+May 08 12:14:43 myserver falco[15442]: grpc: {"created":"@1652026483.513407096","description":"No address added out of total 1 resolved","file":"/build/release/grpc-prefix/src/grpc/src/core/e>
+May 08 12:14:43 myserver falco[15442]: Error starting gRPC server
+```
+
+You may want to run SELinux in 'permissive' mode while you figure out how to give Falco the proper permission to create the gRPC server on startup:
+
+```shell
+sudo setenforce 0
+```
+
+To make this permanent across reboots (if you have Grubby installed):
+
+```shell
+sudo grubby --default-kernel
+/boot/vmlinuz-5.6.13-100.fc30.x86_64
+sudo grubby --update-kernel=/boot/vmlinuz-5.6.13-100.fc30.x86_64 --args='selinux=1 enforcing=0'
+sudo reboot -n
+```
+
+Or if you want, you can turn the permissive mode directly onto the ```/etc/selinux/config``` file:
+
+```shell
+sudo sed -i 's#SELINUX=enforcing#SELINUX=permissive#' /etc/selinux/config 
+sudo cat /etc/selinux/config
+
+# This file controls the state of SELinux on the system.
+# SELINUX= can take one of these three values:
+#     enforcing - SELinux security policy is enforced.
+#     permissive - SELinux prints warnings instead of enforcing.
+#     disabled - No SELinux policy is loaded.
+SELINUX=permissive
+# SELINUXTYPE= can take one of these three values:
+#     targeted - Targeted processes are protected,
+#     minimum - Modification of targeted policy. Only selected processes are protected. 
+#     mls - Multi Level Security protection.
+SELINUXTYPE=targeted
+
+sudo reboot -n
+```
+
+Then check your system logs (for example using ```journalctl --no-page --since 'YYYY-MM-DD' --grep selinux```) and check the messages.
+
```

Comment:
```suggestion

Then restart Falco to verify that it works as expected. To troubleshoot the issue you can then review your system logs (for example using ```journalctl --no-page --since 'YYYY-MM-DD' --grep selinux```). Please refer to the [SELinux manual](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/auditing.md#auditing-selinux-events) for more information about how to read the audit log and correct your SELinux configuration. When done, run:

```shell
sudo setenforce 1
```
to restore enforcing mode.

It is also possible to keep the permissive configuration across reboots if this is necessary to troubleshoot the issue. More information can be found by looking at the examples in the [configuration section in the SELinux manual](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/global_config_files.md#etcselinuxconfig).
```

Reply:
^ here's the suggestion I'm referring to. Note that `sudo setenforce 1` should be in a "shell" snippet exactly like `sudo setenforce 0` but GitHub doesn't let me do that that from a suggestion snippet :man_shrugging: